mods/beds/api.lua

```diff
@@ -91,8 +91,9 @@ function beds.register_bed(name, def)
 			destruct_bed(pos, 1)
 		end,
 
-		on_rightclick = function(pos, node, clicker)
+		on_rightclick = function(pos, node, clicker, itemstack, pointed_thing)
```

Comment:
> In fact 'node' can be removed as it is not used.

Eh no… That's not how lua works. If you leave out `node`, `clicker` will be assigned the node instead and things will go sideways. These aren't named parameters after all. You can only choose to discard variables at the end, but that is quite a bad practice, since that causes people to do things like what the second commit reverts again, which is much more likely to make a speed difference.

But please, elaborate how that is supposed to make the code slower in your opinion. I am interested in it, if it is actually the case. But to my knowledge all arguments are pushed onto the stack anyway, no matter if they are used or not, not that it would be big enough to warrant writing any more about it.